### PR TITLE
Removed parameter-list-no-elements validation errors from redfish modules

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Removed parameter-list-no-elements validation errors from redfish modules

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
idrac_redfish_command.py
idrac_redfish_config.py
idrac_redfish_info.py
redfish_command.py
redfish_config.py
redfish_info.py

### CODE REVIEW

**Comment by felixfontein** [comment]
This means that Ansible now converts elements of these lists to strings (instead of just passing on whatever the user provided).

**Comment by felixfontein** [comment]
I think this should have a changelog fragment.

**Comment by billdodd** [comment]
> This means that Ansible now converts elements of these lists to strings (instead of just passing on whatever the user provided).

For these Redfish modules, the elements in those lists are strings, so this LGTM.

File: changelogs/fragments/1761-redfish-tidy-up-validation.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish modules - explicitly setting lists' elements to `str` (https://github.com/ansible-collections/community.general/pull/1761).
```
Comment (felixfontein) [inline changelogs/fragments/1761-redfish-tidy-up-validation.yml:2]:
```suggestion
  - redfish modules - explicitly setting lists' elements to ``str`` (https://github.com/ansible-collections/community.general/pull/1761).
```

**Review by felixfontein — APPROVED** [review]

**Pull request merged**